adserver/router.py

```diff
@@ -0,0 +1,41 @@
+from .models import AdImpression
+from .models import GeoImpression
+from .models import KeywordImpression
+from .models import PlacementImpression
+from .models import RegionImpression
+from .models import RegionTopicImpression
+from .models import UpliftImpression
+
+
+class IndexRouter:
+    """
+    A database router that allows for indexing into a different postgres DB than core operations.
```

Comment:
Is the plan that this would somehow be optionally enabled somehow?

Reply:
Woops. I had it locally to try and generate reports against prod data but locally (so I could do migrations), but hit issues with the FK. I wonder about denormalizing all the FK data on these reports -- would make them faster, and basically the only thing we're pulling from the advertiser relation is the pricing (CPC/CPM and cost) as well as the campaign type (house/community/paid). Might be useful to just store that on the record.